tests/conftest.py

```diff
@@ -164,6 +165,7 @@ def config_entry(hass):
             "domain": "econnect_metronet",
             "system_base_url": "https://example.com",
         },
+        state=ConfigEntryState.SETUP_IN_PROGRESS,
```

Comment:
This change is related to:
```
Detected code that uses async_config_entry_first_refresh, which is only supported when entry state is ConfigEntryState.SETUP_IN_PROGRESS, but it is in state ConfigEntryState.NOT_LOADED, This will stop working in Home Assistant 2025.11
```

@andreapier do you confirm this happens only in our tests, and not when you run Home Assistant? I want to be sure `async_config_entry_first_refresh` is still the right API to use in our production code. Thanks!

Reply:
Yes, I confirm this only happens in test.
I assume that during config_flow, HA is passing the config entry with the correct state (didn't debug it though).
No warning/error/whatever is logged during config flow so we should be safe.

Reply:
Great!